src/models/IssueModel.js

```diff
@@ -0,0 +1,21 @@
+import {observable} from 'mobx';
+
+export default class IssueModel {
+  store;
+  issue_id;
+	name;
+
+  constructor(store, {issue_id, name}) {
+    this.store = store;
+    this.issue_id = issue_id;
+    this.name = name;
+
+    this.getId = () => {
+      return this.issue_id;
+    }
+
+    this.getName = () => {
```

Comment:
Nahhh, issues will probably be globally defined from the database. I don't think it would change on the front-end.